packages/tiny-graphql-query-compiler/src/index.ts

```diff
@@ -86,7 +86,7 @@ export interface VariableOptions {
   type: string;
   name?: string;
   value?: any;
-  required?: string;
+  required?: boolean;
```

Comment:
This was a mistake -- when I re-added this property to support the metadata coming from the client generator I just got this type wrong, it should be boolean.